### PR TITLE
Adds JSON template for V2.2 manifest schema and container configuration.

### LINKED ISSUE #36: Add json intermediates for manifest and container configuration.
These json intermediates can be used to instantiate real manifest (see #31) and container configuration (see #32) objects.

### LINKED ISSUE #37: Add v1-compatible manifest and layer json.

### PR DESCRIPTION
**Pull request by coollog**
Fixes #36 and fixes #37

### CODE REVIEW

File: crepecake/src/main/java/com/google/cloud/tools/crepecake/json/templates/V21ManifestTemplate.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
+package com.google.cloud.tools.crepecake.json.templates;
```
Comment (elharo) [inline crepecake/src/main/java/com/google/cloud/tools/crepecake/json/templates/V21ManifestTemplate.java]: copyright

File: crepecake/src/test/java/com/google/cloud/tools/crepecake/json/templates/ContainerConfigurationTemplateTest.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
+package com.google.cloud.tools.crepecake.json.templates;
```
Comment (elharo) [inline crepecake/src/test/java/com/google/cloud/tools/crepecake/json/templates/ContainerConfigurationTemplateTest.java]: copyright

File: crepecake/src/test/java/com/google/cloud/tools/crepecake/json/templates/V22ManifestTemplateTest.java [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+/** Tests for {@link V22ManifestTemplate}. */
+public class V22ManifestTemplateTest {
+
+  @Test
+  public void testToJson() throws DigestException, IOException, URISyntaxException {
+    // Loads the expected JSON string.
+    File jsonFile =
+        new File(getClass().getClassLoader().getResource("json/v22manifest.json").toURI());
+    final String expectedJson =
+        CharStreams.toString(new InputStreamReader(new FileInputStream(jsonFile)));
```
Comment (elharo) [inline crepecake/src/test/java/com/google/cloud/tools/crepecake/json/templates/V22ManifestTemplateTest.java]: needs a character set

File: crepecake/src/main/java/com/google/cloud/tools/crepecake/json/templates/V21ManifestTemplate.java (hunk 1 above)
Comment (coollog) [reply crepecake/src/main/java/com/google/cloud/tools/crepecake/json/templates/V21ManifestTemplate.java]: Added copyright

File: crepecake/src/test/java/com/google/cloud/tools/crepecake/json/templates/ContainerConfigurationTemplateTest.java (hunk 2 above)
Comment (coollog) [reply crepecake/src/test/java/com/google/cloud/tools/crepecake/json/templates/ContainerConfigurationTemplateTest.java]: Added copyright

**Pull request merged**